client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -416,7 +416,7 @@ export const SeoForm = React.createClass( {
 		const activateVerificationServices = () => this.props.activateModule( siteId, 'verification-tools' );
 		const isSitePrivate = siteSettings && parseInt( siteSettings.blog_public, 10 ) !== 1;
 		const isJetpackUnsupported = siteIsJetpack && ! jetpackVersionSupportsSeo;
-		const isDisabled = isSitePrivate || isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
+		const isDisabled = isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
```

Comment:
@tyxla updated to work for `hidden` sites in 64c3c64d3c496e25e20865b84dd96d011e2ecb6f. I've also included @roundhill's suggestion about the warning text.